test/StakeManager.js

```diff
@@ -1501,5 +1484,15 @@ describe('StakeManager', function () {
       const tx = stakeManager.connect(signers[10]).delegate(stakerIdAcc4, amount);
       await assertRevert(tx, 'Staker is slashed');
     });
+
+    it('Staker should be able to stake any amount of stake', async function () {
```

Comment:
This test doesn't make sense. Define 'any amount'

Reply:
So , should I rename it as 'Staker should be able to stake less than minStake amount' ?

Reply:
Agreed. Any amount is too vague. We need to test for 0, min stake, < min stake, > min stake ideally.

Reply:
Done @SkandaBhat